oxen/python/oxen/ops/create_label_map.py

```diff
@@ -0,0 +1,17 @@
+import oxen
+
+
+class CreateLabelMap(oxen.Op):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+    def call(self, args):
+        lines = args[0].split("\n")
+        canonical_labels = set(lines)
+        data_labels = set(args[1])
```

Comment:
We might want to add some comments or checks for what arg[0] and arg[1] are expected to be

Reply:
updated!